releasenotes/notes/Instruction-layering-for-circuit-drawing-has-changed-f62ce5aaeb8ce221.yaml

```diff
@@ -0,0 +1,13 @@
+---
+prelude:
+  - |
+    Right/left justification of gates in circuit drawing was incorrect in
+    previous releases.
```

Comment:
The prelude section is actually a prelude for the release not the individual release note. For example see: https://qiskit.org/documentation/release_notes.html#prelude We probably should remove this because we don't want the prelude for the terra 0.10.0 release to say this, because while it's a good fix it's not going to be a headline change of the release.

Reply:
Oops, didn't understand semantics of `reno`. Will fix right now.

Reply:
And the revert was kind of a finger slip :)

Reply:
Changed and pushed release note.

Reply:
@mtreinish I think I'm over my head with reverting my revert to master ... can you fix that, please?

Reply:
I had to do a full rebase locally to unwind it (your change still had the readme) which involved a force push to your branch. But it should be correct now.